WoWAddonUpdater.py

```diff
@@ -39,10 +37,8 @@ def __init__(self):
             confirmExit()
 
         if not isfile(self.INSTALLED_VERS_FILE):
-            with open(self.INSTALLED_VERS_FILE, 'w') as newInstalledVersFile:
-                newInstalledVers = configparser.ConfigParser()
-                newInstalledVers['Installed Versions'] = {}
-                newInstalledVers.write(newInstalledVersFile)
+            open(self.INSTALLED_VERS_FILE, 'w') # Create installed versions file since it doesn't exist
```

Comment:
As with the `with open() as x` syntax, you should not open the file without then closing it. 
i.e. `open(self.INSTALLED_VERS_FILE, 'w').close()`

Reply:
I've left the `with open() as x` unchanged from master. My understanding is that files are automatically closed after exiting the `with open() as x` https://preshing.com/20110920/the-python-with-statement-by-example/